### DISCUSSION TITLE
`logs.persistence.enabled` behaviour

### DISCUSSION THREAD

**Discussion by fernhtls** [apache/airflow#23011] [Q&A]
Hi,

Recently on checking our airflow 2 deployment, using the apache airflow helm chart, I've notice a PV / PVC and disk being created even with the parameter `logs.persistence.enabled` set to `false`.

The PVC is being mounted only in the scheduler, not in any other pod.

I've tried to change the parameter anyway to `true` to setup the disk size and PV / PVC properly, but it seems that the PVC ends up being mounted for both the `scheduler` and `webserver`, what caused some problems for GCP, as we didn't have the disk setup to be able to mount on multiple nodes, and the feature is still on beta / alpha for terraform to apply these changes, and probably the feature itself on GCP.

We are not using the latest helm chart version though, we are using `1.1.0`, but planning to upgrade, but on checking the templates i didn't see any changes regarding the `logs.persistence`.

Are there any plans to change the behaviour?

Thanks,

Fernando

**Comment by potiuk** [comment]
I think its not hapening in the latest chart. It does not look like in the code. But you cand double check it.

**Reply by fernhtls** [reply]
hi @potiuk , I'll give it soon. Thanks!
